.github/shared/package.json

```diff
@@ -20,7 +20,8 @@
     "./swagger": "./src/swagger.js",
     "./tag": "./src/tag.js",
     "./simple-git": "./src/simple-git.js",
-    "./test/examples": "./test/examples.js"
+    "./test/examples": "./test/examples.js",
```

Comment:
I did notice this looks odd. Was this [intentional?](https://github.com/Azure/azure-rest-api-specs/commit/e3131c24003dc8a9485328efa2a4eae004bc46aa) If so I'll continue leaving this alone.